test/tchannel_bridge.js

```diff
@@ -88,17 +91,17 @@ describe ('test tchannel span bridge', () => {
                 entryPoint: path.join(__dirname, 'thrift', 'echo.thrift') // ignored in json case
             });
 
-            // register the server request function.
-            let context = {};
-            encodedChannel.register(server, 'Echo::echo', context, bridge.tracedHandler(handleServerReq));
+            // We pass 'null' as the context because the handler will provide a context
+            // that gets created on every request.
```

Comment:
incorrect comment. I would recommend `let options: any = {}` and pass `options`, to match the semantics of the method signature.